packages/@aws-cdk/aws-servicecatalogappregistry/lib/common.ts

```diff
@@ -0,0 +1,70 @@
+import * as crypto from 'crypto';
+import * as iam from '@aws-cdk/aws-iam';
+
+/**
+ * The options that are passed into a share of an Application or Attribute Group.
+ */
+export interface ShareOptions {
+  /**
+   * Explicitly allow share of application with principals outside of your AWS Organization.
+   *
+   * @default true
+   */
+  readonly allowExternalPrincipals?: boolean;
```

Comment:
not sure if this should be `true` be default...pinging @rix0rrr for advice

Reply:
I revised the docstring to be more clear on this. +1 to @arcrank on this. The default behavior is explained further in step 8 of RAM's resource sharing getting started guide: https://docs.aws.amazon.com/ram/latest/userguide/getting-started-sharing.html

Reply:
I would further explain the behavior in these docs to address @rix0rrr's second point; is there a method that must be called to actually share this resource? This docstring should tell the user not only that this sharing can happen, but how it happens.

Reply:
If the behavior relates to the organization, why don't we call it `allowPrincipalsOutsideOrganization` ?

Reply:
Thanks for the discussion on this. After conferring with the team, this property has been removed from `ShareOptions` as it is always set to `false` in AppRegistry's sharing experience with RAM and not surfaced to the customer.